test/karma.conf.js

```diff
@@ -102,7 +102,7 @@ module.exports = function(config) {
       bs_safari_14: {
```

Comment:
Sorry, one more nitpick, maybe we should make this
```suggestion
      bs_safari_14_0: {
```
for consistency with 13_1, and then later add 14_1 as well?

Reply:
I would just target the latest stable one, since we do the same in Chrome and Firefox. I don't think it'll be fixed to 14.0 for BigSur

Reply:
Oki :+1: